raiden/transfer/state.py

```diff
@@ -528,7 +528,7 @@ def from_dict(cls, data: Dict[str, Any]) -> 'TokenNetworkState':
             serialization.identity,
             data['partneraddresses_to_channelidentifiers'],
         )
-        restored.partneraddresses_to_channelidentifiers = defaultdict(
+        restored.partneraddresses_to_channelidentifiers = defaultdict(  # type: ignore
```

Comment:
What was this complaining about?

Reply:
type mismatch. defaultdict is not a generic.